python/cudf/cudf/utils/utils.py

```diff
@@ -160,16 +162,41 @@ def cudf_dtype_from_pydata_dtype(dtype):
     return infer_dtype_from_object(dtype)
 
 
-def is_single_value(val):
+def is_scalar(val):
     return (
-        isinstance(val, str)
+        val is None
+        or isinstance(val, str)
         or isinstance(val, numbers.Number)
         or np.isscalar(val)
         or isinstance(val, pd.Timestamp)
         or isinstance(val, pd.Categorical)
     )
 
 
+def to_cudf_compatible_scalar(val, dtype=None):
```

Comment:
Likely should go to `utils/dtypes.py` and possibly needs a better name - suggestions, @trxcllnt ?

Reply:
I like this name, it’s simple yet explicit. Is it long? Maybe. Do I immediately know what it’s suppose to do? Yes.